otter/grade/__init__.py

```diff
@@ -129,11 +129,11 @@ def main(
     # Merge dataframes
     output_df = merge_csv(grade_dfs)
     cols = output_df.columns.tolist()
-    output_df = output_df[cols[-1:] + cols[:-1]]
+    output_df = output_df[[KEY_SCORES_DICT_FILE] + cols[:-3] + [KEY_SCORES_DICT_TTL_PTS, KEY_SCORES_DICT_PCT_CRT]]
```

Comment:
rather than hardcoding the index in `cols[:-3]` can we instead do something like

```python
question_cols = sorted(c for c in cols if c not in {KEY_SCORES_DICT_FILE, KEY_SCORES_DICT_TTL_PTS, KEY_SCORES_DICT_PCT_CRT})
output_df = output_df[[KEY_SCORES_DICT_FILE, *question_cols, KEY_SCORES_DICT_TTL_PTS, KEY_SCORES_DICT_PCT_CRT]]
```

Reply:
That is solid. Thanks. I didn't like that hard-coded moment.